Gordon360/ApiControllers/HousingController.cs

```diff
@@ -153,10 +154,12 @@ public IHttpActionResult SaveApplication([FromBody] ApartmentApplicationViewMode
             //get token data from context, username is the username of current logged in person
             ClaimsPrincipal authenticatedUser = this.ActionContext.RequestContext.Principal as ClaimsPrincipal;
             string username = authenticatedUser.Claims.FirstOrDefault(x => x.Type == "user_name").Value;
+            string userID = _accountService.GetAccountByUsername(username).GordonID;
 
             string sessionID = Helpers.GetCurrentSession().SessionCode;
 
-            string editorID = _accountService.GetAccountByUsername(applicationDetails.EditorUsername).GordonID;
+            string editorUsername = applicationDetails.EditorUsername;
```

Comment:
No, they should not be the same...
The `username` corresponds to the current user, which is gotten from the authentication stuff, whereas the `editorUsername` corresponds to the editor that was specified within the View Model data. They should not be assumed to be the same, and this is something that is checked in the service method to prevent suspicious behavior

Reply:
I see the confusion now that I viewed it in the full file diff. The typo is actually the line after this, where I copied and pasted but forgot to change the `username` to `editorUsername`. This must have happen when I was converting all the things from using username back to using student ID numbers.

Reply:
It turns out I made that same mistake in all 3 methods that involve the `editorUsername` (or `newEditorUsername`). That would have caused many problems in the future...

Reply:
Ah, glad I said something!